### PR TITLE
create virt-who org using default_location

### PR DESCRIPTION
**Pull request by hkx303**
Solved issue:
https://github.com/SatelliteQE/robottelo/issues/7632

### CODE REVIEW

File: tests/upgrades/test_virtwho.py [hunk 1]
```diff
@@ -90,6 +91,9 @@ def test_pre_create_virt_who_configuration(self):
             3. Report is sent to satellite.
             4. Virtual sku can be generated and attached.
         """
+        default_loc_id = entities.Location().search(
+            query={'search': 'name="{}"'.format(DEFAULT_LOC)})[0].id
+        entities.Location(id=default_loc_id).read()
```
Comment (vijay8451) [inline tests/upgrades/test_virtwho.py]: Please use below way to use `default_location`
```
default_loc_id =  entities.Location().search(query={'search': 'name="{}"'.format(DEFAULT_LOC)})[0].id
default_loc = entities.Location(id=default_loc_id).read()
org = entities.Organization(name=self.org_name).create() 
default_loc.organization.append(entities.Organization(id=org.id))
default_loc.update(['organization'])
```

Comment (vijay8451) [reply tests/upgrades/test_virtwho.py]: Once you done with it than give try to run test as well ..

Comment (hkx303) [reply tests/upgrades/test_virtwho.py]: Thanks @vijay8451  modified. Test result soon

**Comment by hkx303** [comment]
Added testing result from my end as upgrade-sat-host is busy now

```
(venv) [hkx303@kuhuang upgrades]$ pytest -m 'pre_upgrade' test_virtwho.py 
==================================== test session starts =====================================
platform linux -- Python 3.7.3, pytest-4.6.3, py-1.8.0, pluggy-0.12.0
rootdir: /home/hkx303/Documents/CI/robottelo/robottelo
plugins: services-1.3.1, forked-1.0.2, mock-1.10.4, cov-2.7.1, xdist-1.29.0
collected 2 items / 1 deselected / 1 selected                                                

test_virtwho.py .                                                                      [100%]

====================================== warnings summary ======================================
/home/hkx303/Documents/CI/robottelo/venv/lib/python3.7/site-packages/_pytest/mark/structures.py:337
  /home/hkx303/Documents/CI/robottelo/venv/lib/python3.7/site-packages/_pytest/mark/structures.py:337: PytestUnknownMarkWarning: Unknown pytest.mark.pre_upgrade - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    PytestUnknownMarkWarning,

/home/hkx303/Documents/CI/robottelo/venv/lib/python3.7/site-packages/_pytest/mark/structures.py:337
  /home/hkx303/Documents/CI/robottelo/venv/lib/python3.7/site-packages/_pytest/mark/structures.py:337: PytestUnknownMarkWarning: Unknown pytest.mark.post_upgrade - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    PytestUnknownMarkWarning,

-- Docs: https://docs.pytest.org/en/latest/warnings.html
==================== 1 passed, 1 deselected, 2 warnings in 390.12 seconds ====================
2020-02-10 20:08:01 - robottelo - INFO - Save failed tests to file /var/tmp/robottelo_pre_upgrade_failed_tests.json

```

**Pull request merged**